marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/calculateMergingActivity.api

```diff
@@ -0,0 +1,13 @@
+{
+  "functionName": "calculateMergingActivity",
```

Comment:
I think a "desc" would be very helpful here to describe what will be returned (which has the benefit of then appearing in the generated Java code).